int16 size
rtabmap_ros/KeyPoint[] kptsVec
